src/UI/MainMenuOptions.cpp

```diff
@@ -69,7 +69,6 @@ void MainMenuOptions::init()
     cmbResolution.size(200, 0);
     cmbResolution.maxDisplayItems(1);
 
-    cmbResolution.clearSelection();
```

Comment:
Err, why is this line being removed? This looks out of place.

Reply:
That's one of those auto-commits where if there's no conflict, the change just gets merged without my ability to check it first.

Reply:
But yeah, that line shouldn't have been removed.

Reply:
It exists in #159 so merging this first should ensure it gets put back.

At the same time, now that I think about it, this code is run in the constructor. the resolution list is already going to be empty so clearing it before filling it won't make a difference.

I just tested the code with and without clearing the list first. No difference.

Reply:
This is no longer branched off of #159. It was rebased. It is now an independent branch with parallel development. If both branches modify the same line, it will result in a merge conflict.

More to the point, the change has nothing to do with this commit, nor with this branch even. This branch was about renaming a constant. It's out of place and confusing to have that modification here.